addons/stock_account/models/stock_quant.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class StockQuant(models.Model):
+    _inherit = 'stock.quant'
+
+    value = fields.Float('Value', compute='_compute_value')
```

Comment:
probably missing a groups inventory manager

access right issue when inventory user> 
Sorry, you are not allowed to access documents of type 'Decimal Precision' (decimal.precision). This operation is allowed for the groups: - Administration/Settings - (Operation: read, User: 6)